airflow/operators/docker_operator.py

```diff
@@ -71,7 +74,7 @@ def __init__(
             image,
             api_version=None,
             command=None,
-            cpus=1.0,
+            cpu_shares=None,
```

Comment:
If people are using `cpus` in their code this will break it. If you really think this semantic change is important, you need to deprecate `cpus` by assigning it and warning about eventual deprecation in 2.0.